feat(content): Gegno Content Enhancements (#8147)
endless-sky/endless-sky: data/gegno/gegno outfits.txt

	sound "bludgeon impact"

# Hand to Hand Weapons

outfit "Savagery Pike"
	category "Hand to Hand"

> 	cost 21000

Reviewer: Fragmentation Grenades at 17k and do more attack (1.3 vs 0.9) and have less defense (0.3 vs 0.9), making them much better than this.
```suggestion
	cost 13000
```

	thumbnail "outfit/savagery pike"
	"capture attack" .9
	"capture defense" .9
	"unplunderable" 1

> 	description `This is a ceremonial spear-like weapon presented to Vi warriors when they officially become part of the military. On the forward end are several long, sharp pikes that can easily pierce enemies, and on the rear end is a built-in cannon that can fire up to two explosive rounds. The cannon rounds are rarely used by the Vi unless they deem it absolutely necessary to defeat their foe, or to deal with unanticipated outside factors.`

Reviewer 2: ```suggestion
	description `This is a ceremonial spear-like weapon presented to Vi warriors when they officially become part of the military. On the forward end are several long, sharp pikes that can easily pierce enemies, and on the rear end is a built-in cannon that can fire up to two explosive rounds. The cannon rounds are rarely used by the Vi unless they deem it absolutely necessary to defeat their foe or deal with unanticipated outside factors.`
```

outfit "Irate Carronade"
	category "Hand to Hand"
	cost 39000
	thumbnail "outfit/irate carronade"
	"mass" 3
	"outfit space" -3
	"capture defense" 3.7
	"unplunderable" 1

> 	description `A minaturized version of a ship-to-ship cannon, haphazardly shoved into a small makeshift barricade, isn't exactly the safest practice of installing boarding defenses. Despite this, the hard-armored hulls of larger Gegno warships prove resilient enough to field Irate Carronades. Anyone, or anything, that unknowingly approaches a corridor where one is installed will likely become a new paint layer on the nearby walls.`

Reviewer 3: ```suggestion
	description `A minaturized version of a ship-to-ship cannon, haphazardly shoved into a small makeshift barricade, isn't exactly the safest way to install boarding defenses. Despite this, the hard-armored hulls of larger Gegno warships prove resilient enough to field Irate Carronades. Anyone, or anything, that unknowingly approaches a corridor where one is installed will likely become a new paint layer on the nearby walls.`
```

Author: How about:

"Miniaturizing ship-to-ship cannons and haphazardly shoving them into small makeshift barricades isn't exactly the safest way to install boarding defenses."

Author: I'm not sure if the comments are needed.

Reviewer 3: Sounds good.

	
outfit "Plasmasickle"
	category "Hand to Hand"

> 	cost 17000

Reviewer: ```suggestion
	cost 11000
```

	thumbnail "outfit/plasmasickle"
	"capture attack" .7
	"capture defense" .6
	"unplunderable" 1

> 	description `Plasmasickles are specialized assault weapons that are very reliant on close-quarters combat. Using a small portion of the Scin's recent plasma technology, the blade can be heated to higher temperatures in order to slice through tougher targets. Only select groups of Scin crew are equipped with these, but they are trained well enough to use them effectively.`

Reviewer: ```suggestion
	description `Plasmasickles are weapons specialized for close-quarters combat. Incorporating the Scin's recently developed plasma technology, the blade is heated to high temperatures, enabling it to slice through tough targets. Only select groups of Scin crew are equipped with these, but they are trained well enough to use them effectively.`
```
Maybe?

Author: Yeah, that sounds a lot cleaner/

> 	description `Plasmasickles are specialized assault weapons that are very reliant on close-quarters combat. Using a small portion of the Scin's recent plasma technology, the blade can be heated to higher temperatures in order to slice through tougher targets. Only select groups of Scin crew are equipped with these, but are trained well enough in assassination to use them effectively.`

Reviewer 2: ```suggestion
	description `Plasmasickles are specialized assault weapons that are very reliant on close-quarters combat. Using a small portion of the Scin's recent plasma technology, the blade can be heated to higher temperatures in order to slice through tougher targets. Only select groups of Scin crew are equipped with these, but they are trained well enough in assassination to use them effectively.`
```

outfit "Plasma Grenades"
	plural "Plasma Grenades"
	category "Hand to Hand"

> 	cost 24000

Reviewer: These have superior attack to Korath repeater rifles.
```suggestion
	cost 60000
```

	thumbnail "outfit/plasma grenades"
	"capture attack" 1.9
	"capture defense" 1.2
	"unplunderable" 1

> 	description `With development of more plasma-based weaponry, the Scin invented their own type of grenades using plasma to create powerful explosions. They are effective at clearing out large groups of enemies, and are less of a hazard than unpredictable fragmentation grenades, allowing them to be a bit more reliable for defensive use. These grenades are kept in odd magentic storage devices that fortunately don't accidentally detonate them - except on rare occasions.`

Reviewer 2: ```suggestion
	description `With development of more plasma-based weaponry, the Scin invented their own type of grenades using plasma to create powerful explosions. They are effective at clearing out large groups of enemies and are less of a hazard than unpredictable fragmentation grenades, allowing them to be a bit more reliable for defensive use. These grenades are kept in odd magentic storage devices that prevent them from accidentally detonating - except on rare occasions.`
```

outfit "Irate Carronade"
	category "Hand to Hand"
	cost 39000
	thumbnail "outfit/irate carronade"
	"mass" 3
	"outfit space" -3
	"capture defense" 3.7
	"unplunderable" 1
	description `A minaturized version of a ship-to-ship cannon, haphazardly shoved into a small makeshift barricade, isn't exactly the safest practice of installing boarding defenses. Despite this, the hard-armored hulls of larger Gegno warships prove resilient enough to field Irate Carronades. Anyone, or anything, that unknowingly approaches a corridor where one is installed will likely become a new paint layer on the nearby walls.`
	
outfit "Plasmasickle"
	category "Hand to Hand"
	cost 17000
	thumbnail "outfit/plasmasickle"
	"capture attack" .7
	"capture defense" .6
	"unplunderable" 1

> 	description `Plasmasickles are specialized assault weapons that are very reliant on close-quarters combat. Using a small portion of the Scin's recent plasma technology, the blade is able to be heated to higher temperatures in order to slice through tougher targets. Only select groups of Scin crew are equipped with these, but are trained well enough in assassination to use them effectively.`

Reviewer 4: ```suggestion
	description `Plasmasickles are specialized assault weapons that are very reliant on close-quarters combat. Using a small portion of the Scin's recent plasma technology, the blade can be heated to higher temperatures in order to slice through tougher targets. Only select groups of Scin crew are equipped with these, but are trained well enough in assassination to use them effectively.`
```
I think 'is able to' and 'in order to' was just too much so close to each other. Just a feeling though, so feel free to ignore this.
